### PR TITLE
CORE-12137 Added support for default external messaging route config

### PR DESCRIPTION
**Pull request by owenstanford**
A new configuration section has been added to support external message routing defaults.

### CODE REVIEW

File: data/config-schema/src/main/resources/net/corda/schema/cordapp/configuration/external.messaging/1.0/corda.external.messaging.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
           "name": {
             "type": "string",
             "$comment": "The name of the channels accepts alphanumeric values, underscores, periods, and dashes. Empty strings are not accepted and the max. length is 255 characters. Also topics with a period . or underscore _ could collide in internal data structures, so you are advised to use either but not both.",
```
Comment (vkolomeyko) [inline data/config-schema/src/main/resources/net/corda/schema/cordapp/configuration/external.messaging/1.0/corda.external.messaging.json]:
```suggestion
            "$comment": "The name of the channels accepts alphanumeric values, underscores, periods, and dashes. Empty strings are not accepted and the max. length is 100 characters. Also topics with a period . or underscore _ could collide in internal data structures, so you are advised to use either but not both.",
```

**Review by vkolomeyko — COMMENTED** [review]
LGTM

A minor comment posted.

**Review by filipesoliveira — DISMISSED** [review]
LGTM

**Review by filipesoliveira — APPROVED** [review]
LGTM

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Pull request merged**